Make.config

```diff
@@ -77,11 +77,11 @@ MIN_OSX_BUILD_VERSION=10.14
 MIN_OSX_VERSION_FOR_IOS=10.11
 MIN_OSX_VERSION_FOR_MAC=10.11
 
-IOS_SDK_VERSION=12.2
+IOS_SDK_VERSION=12.4
```

Comment:
12.4? Shouldn't it be `12.3`?

Reply:
Nope - https://developer.apple.com/documentation/xcode_release_notes/xcode_10_3_release_notes?language=objc